dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonHelpers.java

```diff
@@ -384,10 +383,10 @@ protected static String jsonifyTags(String[] tags) {
         return sb.toString();
     }
 
-    protected static boolean checkExtSuffixValidity(KeyValuePair kvp) {
+    public static boolean checkExtSuffixValidity(String k, String v) {
```

Comment:
this method and verifyXValue methods can probably be removed as checking can be done during parsing to put convert fields into forms needed for creating type-specific KeyValuePairs